EnvironmentSetup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Setup script which install the Space Concordia Robotics Software team's development environment. 
+
+# Setup venv
+virtualenv -p `which python3.6` venv
+source venv/bin/activate
+
+# Install Requirements
+pip install -r requirements.txt -r requirements-dev.txt
+
+# Setup python and test it
+python setup.py develop
+
+# Install ROS-Kinetic
+wget https://raw.githubusercontent.com/ROBOTIS-GIT/robotis_tools/master/install_ros_kinetic.sh
+chmod 755 ./install_ros_kinetic.sh
+bash ./install_ros_kinetic.sh
+
+sudo apt install ros-kinetic-rosbridge-suite
+
+# Install camera stuff
+sudo apt-get install ros-kinetic-cv-camera
+sudo apt-get install ros-kinetic-web-video-server
```

Comment:
add the `-y` option so the user doesnt need to confirm `y` when downloading the package